js/base/Exchange.js

```diff
@@ -2682,7 +2682,7 @@ module.exports = class Exchange {
         return undefined;
     }
 
-    parseAccount (account) {
+    typeToAccount (account) {
```

Comment:
usually it's preferred to have the method's name starting with a verb, like `convertTypeToAccount`